CoreFoundation/Base.subproj/CFFileUtilities.c

```diff
@@ -1276,7 +1276,7 @@ CFStringRef _CFXDGCreateRuntimeDirectoryPath(void) {
 }
 
 
-CF_PRIVATE CFArrayRef _CFTokenizeStringToCFArrayOfCFStrings(const char *values, char delimiter) {
+CF_PRIVATE CFArrayRef _CFCreateCFArrayByTokenizingString(const char *values, char delimiter) {
     size_t pathCount = 0;
     char* tmpDirectoriesPaths = (char*)values;
     char* last_colon = 0;
```

Comment:
Sorry to be picky here, but this is code that deals with untrusted input so we have to be extra careful of security concerns.

Can we put a cap on pathCount, as the value causes a stack allocation? Say, 64. If we exceed this count then return a sentinel value, like empty array or NULL (and deal with it one frame up).